modules/searcher/MixCloudSearcher.js

```diff
@@ -0,0 +1,32 @@
+const request = require('request');
+const MixCloudTrack = require('../../models/track/MixCloudTrack');
+
+function MixCloudSearcher() {
+  //
+}
+
+MixCloudSearcher.prototype.search = function(keyword, limit) {
+  let limit = limit || 20;
+  let queryURL = `https://api.mixcloud.com/search/?limit=${limit}&q=${keyword}&type=cloudcast`;
```

Comment:
After I read develop document, I think it don't need auth token, because according docs:
> Access tokens are required to update any data or use the `/me/` shortcut and must only be provided across `https` connections.
Extra information is added to some objects if an access token is provided - for example a `following` attribute is added to a User object to indicate whether or not the authorized user is following the requested User.

So, we just use mixcloud for search music, no update any data or use `/me` api, we limit query in **20**:
```js
let limit = limit || 20;
let queryURL = `https://api.mixcloud.com/search/?limit=${limit}&q=${keyword}&type=cloudcast`;
```